LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-number-expected.txt

```diff
@@ -1,11 +1,11 @@
 
 
-FAIL number: Empty value assert_less_than: expected a number less than 148 but got 148
-FAIL number: Auto width and auto height with a long text assert_greater_than: expected a number greater than 148 but got 148
+FAIL number: Empty value assert_equals: expected 19 but got 23
```

Comment:
The number tests fail more than others due to the height being different between field-sizing: fixed vs content. This seems to be due to the spin buttons being taller than the input is normally.

This is either a bug in the implementation of field-sizing (doubtful given number inputs are ultimately just textcontrols), a bug in the test, or a bug in WebKit's number input styling in general. Will investigate and try to solve in a follow up.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=269177